### ISSUE TITLE
Day-of-week highlight is off by one 

### ISSUE THREAD

**Issue by rmosolgo** [strukturart/greg#42]
Hi! I recently "upgraded" from Android to KaiOS and I found that the built-in calendar wouldn't sync my Google account. I was so glad to find greg -- it's just what I was looking for. Thank you!

Today I noticed that the day-of-week highlight in the calendar view doesn't line up with the selected day for me. Here are a couple of pictures of my device:

![Photo on 3-27-24 at 4 11 PM](https://github.com/strukturart/greg/assets/2231765/8c70d91d-f55f-428d-9644-89cae8edf967)

Above, the cursor is on a Friday, but "Thu" is highlighted in blue at the top row. 

![Photo on 3-27-24 at 4 10 PM](https://github.com/strukturart/greg/assets/2231765/be916391-ffc2-4911-89f4-949c36590dda)

Above, the cursor is on a Wednesday, but "Tue" is highlighted in blue.

We recently started daylight savings time here (USA Eastern time), maybe that's the culprit? 

I have also noticed -- but I don't have pics yet -- that full-day events were showing up on the day _before_ they were actually scheduled (eg, an event for March 27 on Google Calendar would appear on March 26 in greg). I wonder if it's related. I'll open a new issue for that when I have some pictures. 

I have a web development background and I'm interested in learning more about KaiOS development. If you'd like me to take a try at fixing this bug, could you point me to any relevant code in this repo? Thanks!

**Comment by strukturart** [comment]
Thanks for reporting the bug. Do you have the option to sideload the app? then you could install the latest version.

**Comment by rmosolgo** [comment]
I haven't done any of the development setup yet, so I don't think I can sideload it yet... I'll look into it soon!

**Comment by strukturart** [comment]
If you send me the IMEI of your devices via email, I can add you in the store as an app tester, so you will receive the latest version of the app. Otherwise, you can also read up here: https://wiki.bananahackers.net/ you will find out there Whether your device supports sideloading and how you do it. If you want to write your own apps, sideloading is recommended.

**Comment by rmosolgo** [comment]
Yeah, I've been meaning to do it, thanks. I'm on a Nokia 2720 V Flip from Verizon and it seems like it's a bit locked down: after entering debug mode (I've got the bug icon) and downloading `adb`, and hooking it up by USB, `adb` says it's unauthorized. I see there are some more steps to try here but I haven't tried them yet. 

I'm on Mac, so I downloaded Waterfox Classic and got WebIDE going, but it can't see the device yet -- because I need `adb forward ...` to work first, I think.

I'll keep working on it!
